docs/src/main/sphinx/functions/list-by-topic.md

```diff
@@ -477,6 +477,7 @@ For more details, see {doc}`string`
 - {func}`luhn_check`
 - {func}`normalize`
 - {func}`position`
+- {func}`random_string`
```

Comment:
This should be a catalog function, not a global function.

Reply:
Do you have some examples of catalog functions?

Reply:
Take a look at `getFunctions()`, `listFunctions()`, `getFunctionMetadata()`, etc. in `ConnectorMetadata`

Reply:
Being removed in https://github.com/trinodb/trino/pull/23979 and I removed all references in this PR too.